docs/getting-started/index.md

```diff
@@ -0,0 +1,250 @@
+
+   ```shell
+   sudo apt install openjdk-<version>-jdk-headless
+   ```
+
+   Verify the Java version with:
+
+   ```shell
+   java -version
+   ```
+
+   You should see information about the OpenJDK version.
+
+### GCP
+
+To work on the GCP platform, follow these steps:
+
+1. In the Google Cloud console, launch a new instance.
+   Select `e2-standard-4` as the instance type and 20 GB for the boot disk size.
+   Allow HTTP and HTTPS traffic if you want to connect to the instance remotely.
+   Leave all other settings as their defaults.
+   Other operating systems and instance types may work, but are not fully tested.
+
+1. Start the instance and connect to it via the SSH-in-browser tool.
+
+1. Update the Debian OS to ensure it's up-to-date:
+
+   ```shell
+   sudo apt update
+   sudo apt upgrade
+   ```
+
+   You may need to reboot the instance for all changes to take effect.
+
+1. Install the Java Development Kit (JDK), Java 8, 11 and 17 are supported.
+
+   ```shell
+   wget -O - https://apt.corretto.aws/corretto.key | sudo gpg --dearmor -o /usr/share/keyrings/corretto-keyring.gpg
+   echo "deb [signed-by=/usr/share/keyrings/corretto-keyring.gpg] https://apt.corretto.aws stable main" | sudo tee /etc/apt/sources.list.d/corretto.list
+   sudo apt-get update
+   sudo apt-get install -y java-<version>-amazon-corretto-jdk
+   ```
+
+   Verify the Java version with:
+
+   ```shell
+   java -version
+   ```
+
+   You should see information about the OpenJDK version.
+
+### Local workstation
+
+To build and install Gravitino locally on a macOS or a Linux workstation,
+follow these steps:
+
+1. Install the Java Development Kit (JDK), Java 8, 11 and 17 are supported.
+   This can be done using [sdkman](https://sdkman.io/), for example:
+
+   ```shell
+   sdk install java <version>
+   ```
+
+   You can also use different package managers to install JDK, for example,
+   [Homebrew](https://brew.sh/) on macOS, `apt` on Ubuntu/Debian, and
+   `yum` on CentOS/RedHat.
+
+## Install Gravitino
+
+You can install Gravitino from the binary release packages or the container images.
+Follow [how-to-install](../how-to-install.md).
+
+Or you can install Gravitino from scratch.
+Follow [how-to-build](../how-to-build.md) and [how-to-install](../how-to-install.md).
+
+## Start Gravitino
+
+Start Gravitino using the `gravitino.sh` script:
+
+```shell
+<path-to-gravitino>/bin/gravitino.sh start
+```
+
+## Install Apache Hive
+
+If you already have Apache Hive and Apache Hadoop in your environment,
+you can skip this step and use the existing service with Gravitino.
+Or else, you can follow the [instructions](./hive.md) to install Apache Hive.
+
+## Interact with Apache Gravitino API
+
+After having deployed the Gravitino server, you can interact it using
+the RESTful APIs to create and modify metadata.
+
+:::tip
+The following examples use `localhost` as the host name.
+You may need to revise it accordingly based on the environment
+you are using.
```

Comment:
This is too verbose. Perhaps just "You may need to revise it based on your environment."

Reply:
Okay. Will revise.